app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -23,7 +23,7 @@ def index
       result = BackupCodeSetupForm.new(current_user).submit
       visit_result = result.to_h.merge(analytics_properties_for_visit)
       analytics.backup_code_setup_visit(**visit_result)
-      attempts_api_tracker.mfa_enroll_backup_code(success: result.success?)
+      attempts_api_tracker.mfa_enrolled(success: result.success?, mfa_device_type: 'backup_code')
```

Comment:
Should we use `TwoFactorAuthenticatable::AuthMethod::BACKUP_CODE` and friends instead of strings? I don't feel strongly and am not sure whether they line up with what we'd want to send, but figured it was worth mentioning.

Reply:
updated in https://github.com/18F/identity-idp/pull/12163/commits/3ccc0e0ddc01a679f2e75fbd9a23f9ab501a1ea6